### PR TITLE
Updated documentation and cleaned up some parameters...

### PR DESCRIPTION
**Pull request by ibmatc**
Updated documentation and cleaned up some parameters in parameter contexts.  Added example json for Clinical Ingestion flow.

Signed-off-by: Adam T. Clark <atclark@us.ibm.com>

### CODE REVIEW

File: clinical-ingestion/README.md [hunk 1]
```diff
@@ -22,3 +22,22 @@ In addition, certain components in the Clinical Ingestion Flow require specific
...
+
+
+# Running a FHIR bundle through the Clinical Ingestion Flow
+
+There are two ways to easily run a FHIR bundle through Clinical Ingestion Flow once it is deployed to NiFi.
+
+1. Post the FHIR bundle json to the defined port (FHIR_LISTEN_PORT in "Clinical Ingestion Flow FHIR Parameter Context")
+	* curl -X POST --header "ResolveTerminology: true" --header "Content-Type: application/json" -d @<<path/to/json>> "<<NIFI_SERVER>>:<<FHIR_LISTEN_PORT>>/fhirResource" --verbose
+	* "ResolveTerminology: true" tells the pipeline to run the bundle through the terminology normalization process. If you do not wish to run this step, you can omit this header.
+	* "path/to/json" can refer to any FHIR bundle you wish to process. For example, "patientData/fhir_bundle_birthsex.json"
```
Comment (luisgarcc) [inline clinical-ingestion/README.md:34]: The name of this bundle is a bit weird, maybe something like:

patient-with-us-core-birhtsex.json

File: clinical-ingestion/README.md [hunk 2]
```diff
@@ -22,3 +22,22 @@ In addition, certain components in the Clinical Ingestion Flow require specific
...
+	* NIFI_SERVER and FHIR_LISTEN_PORT should be known values from the setup above.
+	* The result of this command should be an HTTP 200 response indicating that it was successfully submitted.
+
+2. Submit the FHIR bundle to the kafka topic configured above (kafka.topic.in in "Clinical Ingestion Flow Kafka Parameter Context")
+	* Using the configured kafka broker (), post the FHIR bundle of your choice and the Clinical Ingestion Flow will automatically react and begin processing.
```
Comment (luisgarcc) [inline clinical-ingestion/README.md:39]: Empty parenthesis after broker ()

**Review by luisgarcc — APPROVED** [review]
Only some minor comments. LGTM

File: clinical-ingestion/README.md (hunk 2 above)
Comment (ibmatc) [reply clinical-ingestion/README.md]: good catch. Forgot to go back and fill that in. Fixed.

**Pull request merged**